models/filtermodel.cpp

```diff
@@ -35,6 +37,34 @@ void FilterModel::setSourceModel(DocumentListModel *model)
     QSortFilterProxyModel::setSourceModel(static_cast<QAbstractItemModel*>(model));
 }
 
+int FilterModel::sortParameter() const
+{
+    return m_sortParameter;
+}
+
+void FilterModel::setSortParameter(int sortParameter)
+{
+    m_sortParameter = sortParameter;
```

Comment:
For minor detail this could have the common change check, if (m_sortParameter == sortParameter) return;

Reply:
Done

Reply:
Done